compliance/github/repository_branch_protection/CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Changelog
 
+## v3.0
+
+- Use GitHub GraphQL API to decrease the number of API calls
+- Remove "branches" parameter and check the protection on the default branch
```

Comment:
Since this is a Policy Template that is published to customers, I don't know if removing functionality like this is really a good idea.

Reply:
I added back the functionality.